src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems.arm;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.trajectory.TrapezoidProfile.Constraints;
+
+public class ArmConstants {
+  public static final int k_armMotorID = 18;
+  public static final int k_smartCurrentLimit = 50;
+  public static final double k_armSpeed = 0.7;
+  public static final double k_climbSpeed = 0.3;
+  public static final double k_speedZero = 0.0;
+  public static final boolean k_isInverted = false;
+
+  public static final double k_armGearRatio =
+      (1.0 / 60.0)
+          * (28.0 / 50.0)
+          * (16.0 / 64.0); // was 1/25, but another maxplanetary was added to make it 1/125.
+  public static final double k_positionConversionFactor = k_armGearRatio;
+  public static final double k_velocityConversionFactor = k_armGearRatio / 60.0;
```

Comment:
Why divided by 60?

Lots of magic numbers in this file.  Would be good to describe how some of them were found.

Reply:
I believe the 60 is simply to convert it to velocity, ergo 60 seconds in a minute. I'll put a comment for this.